### ISSUE TITLE
Bug: when yanking text, moves to the top of the file

### ISSUE THREAD

**Issue by chrisgrieser** [Mofiqul/trld.nvim#7]
I am very confused by this bug. Apparently, as soon as I have this plugin enabled, yanking any text moves me to the top of the file?

**Comment by santigo-dev** [comment]
Maybe there's something mapped to your `y`(you can check with `:map y`), but I don't think that's related to the plugin, at least I don't get that with this plugin nor with my fork of it.

**Comment by chrisgrieser** [comment]
yeah, that's what I thought as well, however disabling nothing but this plugin fixed the issue, strangely 🤔 

So after a bunch of bisecting, I actually found the issue. I use this bit to get a "sticky" yank command. By its own, this snippet works completely fine, but combined with trld.nvim, this snippet causes the bug for me?

```lua
vim.api.nvim_create_autocmd({ "CursorMoved", "VimEnter" }, {
	callback = function() vim.g.cursorPreYank = vim.api.nvim_win_get_cursor(0) end,
})

vim.api.nvim_create_autocmd("TextYankPost", {
	callback = function()
		if vim.v.event.operator == "y" then vim.api.nvim_win_set_cursor(0, vim.g.cursorPreYank) end
	end,
})
```
